lib/serializer.js

```diff
@@ -13,6 +13,24 @@ try {
     safeJsonStringify = null;
 }
 
+// when looping through arbitrary fields attached to the error object, cross
+// reference them against this known list of fields.
+var KNOWN_FIELDS = [
```

Comment:
What happens if verror decides to update its internal fields? Can we reflectively iterate through them instead?